(Lobster) Mapping Catch-up

CawsForConcern opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
This is a general mapping feature pass.

Specific changes to this map:
- Added a button for the radiation shutters for in the Singularity Chamber.
- Added a public fax machine in the Library.
- Added a docking arm that can accommodate both the Mining and Salvage shuttles.
- Changed the layout of the directional windows in Medbay. Hopefully people will finally stop breaking them roundstart (and they shouldn't bug out and prevent you from buckling people into them).

Feature-catch up:
- [x] Nanomed Civilians mapped from #2092
- [x] AI Lawboard crate from #1507
- [x] Surgery & Robotics doors from #1520
- [x] HV & Pipe dock support for #1727
- [x] New furniture from #2050

QoL:
- [x] Ensure HOP has a document printer mapped
- [x] Ensure Medical has a document printer mapped
- [x] Ensure Library has a document printer and fax mapped
- [x] Ensure Chemistry has Cell Recharger mapped
- [x] Ensure Brigmedic has Cell Recharger mapped

Bugfixes:
- [x] Correct EVA Doors from #1741
- [x] Double check that Lawyer + SEC airlocks are properly used (for IAA, Magistrate)
## Why we need to add this

<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
- Nanomed Vending machines were asked to be mapped
- Implementing the AI Lawboard crate makes it so that maps don't need to be updated every time we change the lawboards
- Surgery and Robotics airlocks were added a while ago, this map pass just ensures they're actually being used
- EVA doors are wrong on some maps (some maps use HOP airlocks on EVA), and using EVA doors makes it easier to adjust accesses without updating maps
- The new furniture looks cool and gives Starlight maps some more visual differences from other forks
- HV cables (and pipes at Cargo Docks) need to be added across all maps to support shuttles that will use these features

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

<img width="977" height="551" alt="image" src="https://github.com/user-attachments/assets/446a6b51-b894-4e9a-8b04-19eab0a33eec" />

fig. 1 - Nanomeds mapped

<img width="766" height="633" alt="image" src="https://github.com/user-attachments/assets/e52c1a71-a6de-4130-b0a8-0feef3c202c1" />

fig. 2 - AI lawboards crate

<img width="961" height="701" alt="image" src="https://github.com/user-attachments/assets/bc341162-e7e2-484a-b879-c1b570506553" />
<img width="946" height="730" alt="image" src="https://github.com/user-attachments/assets/c5dcf481-c13f-48f9-81b1-ff68dbc3bd0a" />

fig. 3 - Surgery and Robotics airlocks mapped

<img width="801" height="500" alt="image" src="https://github.com/user-attachments/assets/11569a4c-0e15-4ab2-b603-8c9476c1222a" />
<img width="654" height="511" alt="image" src="https://github.com/user-attachments/assets/77157ef5-82da-4303-965c-2e5e09fdab95" />

fig. 4 - HV cables (and pipes for Cargo only) mapped to all docks (using hex color #F2E408FF from #1745)

<img width="707" height="710" alt="image" src="https://github.com/user-attachments/assets/43a468f4-d688-41ea-b754-4c2cbcdd8329" />
<img width="625" height="614" alt="image" src="https://github.com/user-attachments/assets/8a86d932-6fc7-431c-9386-043f5c8f40a3" />
<img width="1173" height="615" alt="image" src="https://github.com/user-attachments/assets/65cb118e-a9fb-452c-b652-c2bc5f4c5c87" />
<img width="803" height="663" alt="image" src="https://github.com/user-attachments/assets/f5b6b41d-a3df-4543-9cb7-6c928cb9563a" />
<img width="937" height="735" alt="image" src="https://github.com/user-attachments/assets/ecaab58c-bce0-44a8-899a-fdc3a6b5933a" />
<img width="764" height="703" alt="image" src="https://github.com/user-attachments/assets/ddadc6e1-b6b4-4d67-85cc-6d2626fe2848" />

fig. 5 - Some examples of new furniture mapped (not exhaustive, but generally went around and replaced old furniture with couches / benches / etc. where it would make sense and look good. Mostly Bars, Bridges, Offices, Evac, Arrivals.

<img width="587" height="420" alt="image" src="https://github.com/user-attachments/assets/c79a8545-dd8e-48e9-b21b-d159a8417b20" />

fig. 6 - EVA Airlock door is correct

<img width="700" height="543" alt="image" src="https://github.com/user-attachments/assets/ad0179ad-2d50-46dd-945a-f2f6b1e43bd9" />
<img width="699" height="564" alt="image" src="https://github.com/user-attachments/assets/0c224944-64ce-4991-9263-0c619b2aa1f8" />
<img width="624" height="608" alt="image" src="https://github.com/user-attachments/assets/7aa4cbb9-0ecd-4694-b143-5dc6d84360c4" />

fig. 7 - Document printers at HOP, Medical, Library

<img width="1090" height="808" alt="image" src="https://github.com/user-attachments/assets/5613ba64-50fe-4ea0-a3eb-74a102664c09" />

fig. 8 - Brig doors for most access into SEC, SEC/Lawyer doors on Interrogation Room. Generally, someone with lawyer access should always be able to get to GenPop and the Interrogation Room

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: CawsForConcern
- add: (Lobster) New furniture! Updated benches, chairs, sofas added at select locations
- add: (Lobster) AI Lawboard crate (replaces old loose lawboards)
- add: (Lobster) Robotics access doors in Science
- add: (Lobster) Surrgery access doors in Medical
- add: (Lobster) HV cables to shuttle docks
- add: (Lobster) Gas pipes to cargo shuttle docks
- add: (Lobster) Cell Recharger to Chemistry
- add: (Lobster) Cell Recharger to Brigmedic
- add: (Lobster) Document printer to Library
- add: (Lobster) Fax machine to Library
- add: (Lobster) Document printer to Medical
- add: (Lobster) Crew records computer to HOP
- add: (Lobster) Button to open/close the radiation shutters in the Singularity Chamber
- add: (Lobster) Docking arm for Mining and Salvage ships to use
- add: (Lobster) Atmospherics now has O2/N2 miners
- add: (Lobster) Atmospherics now has a condesnor
- add: (Lobster) Atmospherics now has N20 canister
- tweak: (Lobster) Medical Bay directional windows moved so they don't interfere with buckling patients
- fix: (Lobster) Brig / Lawyer doors in the right places for SEC Department

CawsForConcern committed:
- add: (Lobster) New furniture! Updated benches, chairs, sofas added at select locations
- add: (Lobster) AI Lawboard crate (replaces old loose lawboards)
- add: (Lobster) Robotics access doors in Science
- add: (Lobster) Surrgery access doors in Medical
- add: (Lobster) HV cables to shuttle docks
- add: (Lobster) Gas pipes to cargo shuttle docks
- add: (Lobster) Cell Recharger to Chemistry
- add: (Lobster) Cell Recharger to Brigmedic
- add: (Lobster) Document printer to Library
- add: (Lobster) Document printer to Medical
- add: (Lobster) Crew records computer to HOP
- add: (Lobster) Button to open/close the radiation shutters in the Singularity Chamber
- fix: (Lobster) Brig / Lawyer doors in the right places for SEC Department

CawsForConcern committed:
furniture tweaks

CawsForConcern committed:
Updated docking arm

CawsForConcern commented:
Docking arm added, works regardless of where Salvage and Mining dock.
<img width="426" height="388" alt="image" src="https://github.com/user-attachments/assets/b3c4a80b-17b9-4a0a-b67a-e1d46b33af26" />
<img width="477" height="411" alt="image" src="https://github.com/user-attachments/assets/9662106f-5b7b-4559-82a0-76e59f4635a6" />

CawsForConcern commented:
how the hell did this get merge conflicts

CawsForConcern committed:
Merge branch 'starlight-dev' into MapPassLobster

# Conflicts:
#	Resources/Maps/_Starlight/Stations/Lobster.yml

Happyrobot33 commented:
Needs O2/N2 miners to match the PR that was merged https://github.com/ss14Starlight/space-station-14/pull/2128

CawsForConcern committed:
- add: (Lobster) Atmospherics now has O2/N2 miners
- add: (Lobster) Atmospherics now has a condesnor
- add: (Lobster) Atmospherics now has N20 canister

CawsForConcern commented:
> Needs O2/N2 miners to match the PR that was merged #2128

Re-implemented, comparisons below. #2128 was kinda sloppy and changed stuff it didn't mention.

Before any changes from #2128 (just for reference):
<img width="1354" height="1000" alt="image" src="https://github.com/user-attachments/assets/d08a2d6b-4dc8-4049-9100-1e492fa8d281" />

Here's #2128:
<img width="1356" height="1000" alt="image" src="https://github.com/user-attachments/assets/1a6f7648-4e60-4b1a-9ee2-eb6c9f072f2e" />

notes:
- no lights inside of the new O2 and N2 chambers
- no lights on the right side of Atmos (it removed a post light and added none to replace it)
- the waste pipe now dead-ends for no reason
- wrong atmos markers in the chambers (vacuum instead of O2/N2 markers)
- for some reason it removed the disposal bins hooked up to the burn chamber? not mentioned in the changelog
- removed a camera and didn't replace it

This PR:
<img width="1356" height="959" alt="image" src="https://github.com/user-attachments/assets/d267c471-17f9-4aa5-b62e-6a72555b1ec4" />

notes:
- there's lights now
- the waste pipe no longer dead ends like the original atmos
- the atmos markers in the chambers are correct now
- the disposal bins to the burn chamber weren't removed for no reason
- the camera was moved (it's hard to see but it's on the same tile as the condenser
notes:
- there's actually lights
- put in a default waste pipe (there was

Happyrobot33 committed:
Merge branch 'starlight-dev' into MapPassLobster
